### ISSUE TITLE
Syntax errors on older ruby version

### ISSUE THREAD

**Issue by newbieeeeeee** [senny/sablon#124]

**Comment by newbieeeeeee** [comment]
Hi I'm using ruby 1.9.3 and I'm using sablon latest version. I'm getting syntax errors while using it because of the older ruby version obviously. I've tried to fixed it and successfully did it with no errors and used it to create word document. Recently I needed to print HTML with list so I tried but couldn't make it work because of this error. 
ArgumentError: Could not find w:abstractNum definition for style: ''

Obviously its because of the changes I made for removing syntax errors and couldn't fix it.

I've tested latest version of gem in a new project and it worked fine with list. So I decided to add the latest version into my current project but having syntax errors again. I need help to fix the syntax errors because If I do it it would eventually end up in breaking something like last time. These are the syntax errors I'm having:

SyntaxError: 
/lib/sablonnn/configuration/configuration.rb:17: syntax error, unexpected tPOW
    def register_html_tag(tag_name, type = :inline, **options)
                                                      ^
/lib/sablonnn/configuration/configuration.rb:18: syntax error, unexpected tPOW, expecting ')'
      tag = HTMLTag.new(tag_name, type, **options)
                                          ^
/lib/sablonnn/configuration/configuration.rb:56: unknown type of %string
...s: :root, allowed_children: %i[_block _inline] },
...                               ^
/lib/sablonnn/configuration/configuration.rb:56: syntax error, unexpected $end
...s: :root, allowed_children: %i[_block _inline] },

P.s. I can't upgrade my ruby version.

Thanks in advance

**Comment by stadelmanma** [comment]
@newbieeeeeee Typically a missing `w:abstractNum` style is an issue with the template not the gem itself. Sablon makes no effort to try and create some of the complex internals of MS Word and expects them to already exist in the template. Judging by the syntax errors listed I wouldn't think correcting them would break the code that processes HTML lists without breaking everything else.

If you still have your code modified to work with v1.9.3  I would try it again, however this time save an unordered list into the template document somewhere. Alternatively you could try using this template https://github.com/senny/sablon/blob/master/test/fixtures/insertion_template.docx with a context containing lists. That template document has all the required Word styles so if your code is correct it should work.

**Comment by newbieeeeeee** [comment]
@stadelmanma Thanks for your help. I've fixed the syntax issues successfully. And it's generating word file. Although I got a new error which is: 

ArgumentError: Could not find w:abstractNum definition for style: "ListBullet"

Which I successfully resolved too. After a lot of debugging I came to know that the version of the word template I was using was old. So I created a new template with same content and it worked.

Thanks again for your help.
